inspirehep/utils/record_getter.py

```diff
@@ -55,9 +55,9 @@ def wrapper(*args):
         try:
             return f(*args)
         except Exception as e:
-            current_app.logger.error('Error in retrieving record {0} '
-                                     'caused by {1}'.format(repr(args),
-                                                            repr(e)))
+            current_app.logger.error(
+                'Error in retrieving record %s caused by %s',
```

Comment:
Which in term will be the same thing as calling `logger.exception('Can't load recid %s', args)`. This thing will also give more context to the exception so that we filter out `PID` related things and not found errors from actual exceptions.